src/abstractdataframe/show.jl

```diff
@@ -361,9 +356,8 @@ NOTE: The value of `maxwidths[end]` must be the string width of
 - `allcols::Bool = false`: Whether to print all columns, rather than
   a subset that fits the device width.
 - `splitcols::Bool`: Whether to split printing in chunks of columns fitting the
-  screen width rather than printing all columns in the same block. Unless
-  `allcols==true`, all columns will be omitted if any single chunk overflows
-  the screen width. 
+  screen width rather than printing all columns in the same block. By default
+  this is the case only if `io` has the `IOContext` property `limit` set.
```

Comment:
this is not true for this method, it takes `false` as default (as opposed to what `show` does)